macros/src/main/scala/gestalt/macros/DefMacros.scala

```diff
@@ -2,30 +2,34 @@ import scala.gestalt._
 
 object plusObject {
   def apply(a: Int, b: Int): Int = meta {
+    import toolbox._
     q"$a + $b"
   }
 
   def defaultArgs(a:Int, b: Int = 1): Int = meta {
+    import toolbox._
     q"$a + $b"
   }
   def curried(a:Int)(b: Int): Int = meta {
+    import toolbox._
     q"$a + $b"
   }
   def poly(a: Any, b: Int): Int = meta {
     import toolbox._
     a match {
       case Lit(i:Int) => q"$a + $b"
-      case Lit(s:String) => q"$a.toInt + $b"
+      case Lit(s:String) => q"${a.wrap}.toInt + $b"
```

Comment:
We already have implicits to automatically lift it, that's why it's not required in other places.

Here it's required due to ambiguity about the typed and untyped constructors: `.toInt` could be thought as both typed or untyped, but only the untyped version type checks.